features/features.go

```diff
@@ -25,6 +25,7 @@ type Features struct {
 	NoImplicitIgnore           bool `long:"no-implicit-ignore" description:"disable implicit ignore rules to exclude .tmp-earthly-out/, build.earth, Earthfile, .earthignore and .earthlyignore when resolving local context"`
 	CheckDuplicateImages       bool `long:"check-duplicate-images" description:"check for duplicate images during output"`
 	EarthlyVersionArg          bool `long:"earthly-version-arg" description:"includes EARTHLY_VERSION and EARTHLY_BUILD_SHA ARGs"`
+	ExplicitGlobal             bool `long:"explicit-global" description:"require base target args to have explicit settings to be considered global args"`
```

Comment:
Mind enabling this in `0.7`? Look for `case versionAtLeast(ftrs, 0, 7):` in this file.

Reply:
sure, so just to confirm, keep it as a feature flag for 0.6 but enable by default for 0.7?
Would it require a separate test file which will use 0.7?

Reply:
Yes, exactly. No additional test needed for 0.7 for now.